lib/std/c/freebsd.zig

```diff
@@ -622,6 +622,11 @@ pub const MAP = struct {
     pub const NOCORE = 0x00020000;
     pub const PREFAULT_READ = 0x00040000;
     pub const @"32BIT" = 0x00080000;
+
+    pub fn ALIGNED(a: i32) u32 {
```

Comment:
Types are mismatched here: in is `i32` and out is `u32`, so when this is actually referenced it will fail to compile. Also, is it guaranteed for `a` to be `i32`? Also, what is `a` meant to be here?

Reply:
good point actually u32 would be more appropriate.